steering_controllers_library/src/steering_controllers_library.cpp

```diff
@@ -81,27 +81,103 @@ controller_interface::CallbackReturn SteeringControllersLibrary::on_configure(
   const rclcpp_lifecycle::State & /*previous_state*/)
 {
   params_ = param_listener_->get_params();
+
+  // TODO(anyone): Remove deprecated parameters
+  // START OF DEPRECATED
+  if (!params_.front_steering)
+  {
+    RCLCPP_WARN(
+      get_node()->get_logger(),
+      "DEPRECATED parameter 'front_steering', set 'traction_joint_names' or 'steering_joint_names' "
```

Comment:
Parameter is actually called "traction_joints_names".

Reply:
Fixed.